apps/company/tasks.py

```
@@ -0,0 +1,338 @@
+    recommendations = []
+    
+    for sale in products:
+
+        product = sale['vendor_code']
+        
+        product = Product.objects.filter(vendor_code=product)
+        product_w = product.filter(marketplace_type="wildberries")
+        if product_w.exists():
+            product = product_w.first()
+        else:
+            product = product.first()
```

Ruff

Use ternary operator `product = product_w.first() if product_w.exists() else product.first()` instead of `if`-`else`-block

Replace `if`-`else`-block with `product = product_w.first() if product_w.exists() else product.first()`

(SIM108)